components/NearestBloodBanksList.client.vue

```
@@ -0,0 +1,89 @@
+function currentOrigin() {
+  const hasPendingSearch =
+    mapCenter.value[0] !== initialCenter[0] ||
+    mapCenter.value[1] !== initialCenter[1];
+  return hasPendingSearch ? mapCenter.value : userLocation.value;
+}
+
+function refresh() {
+  const origin = currentOrigin();
+  if (!origin) return;
+
+  const sorted = sortPointsByDistance(mapStore.pinMarkersFeatures, origin, 30);
```

ESLint

[error] 'sortPointsByDistance' is not defined.

(no-undef)